### ISSUE TITLE
CCE: class `org.slf4j.helpers.NOPLogger` cannot be cast to class `ch.qos.logback.classic.Logger` when running Diktat as a Gradle plug-in

### ISSUE THREAD

**Issue by 0x6675636b796f75676974687562** [saveourtool/diktat#1575]
## Symptoms

```
SLF4J: No SLF4J providers were found.
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.
SLF4J: Class path contains SLF4J bindings targeting slf4j-api versions 1.7.x or earlier.
SLF4J: Ignoring binding found at [jar:file:/home/runner/.gradle/caches/modules-2/files-2.1/ch.qos.logback/logback-classic/1.2.11/4741689214e9d1e8408b206506cbe76d1c6a7d60/logback-classic-1.2.11.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See https://www.slf4j.org/codes.html#ignoredBindings for an explanation.
Error: Exception in thread "main" java.lang.ClassCastException: class org.slf4j.helpers.NOPLogger cannot be cast to class ch.qos.logback.classic.Logger (org.slf4j.helpers.NOPLogger and ch.qos.logback.classic.Logger are in unnamed module of loader 'app')
	at com.pinterest.ktlint.internal.KtlintCommandLine$configureLogger$2.invoke(KtlintCommandLine.kt:253)
	at com.pinterest.ktlint.internal.KtlintCommandLine$configureLogger$2.invoke(KtlintCommandLine.kt:252)
	at com.pinterest.ktlint.core.KtLintKLoggerInitializerKt.initKtLintKLogger(KtLintKLoggerInitializer.kt:42)
	at com.pinterest.ktlint.internal.KtlintCommandLine.configureLogger(KtlintCommandLine.kt:259)
	at com.pinterest.ktlint.internal.KtlintCommandLine.run(KtlintCommandLine.kt:209)
	at com.pinterest.ktlint.Main.main(Main.kt:31)
```

## Versions affected

| Diktat | KtLint | io.github.microutils:kotlin-logging | SLF4J API versions | Affected? |
|--------|--------|--------|--------|--------|
| 1.2.4 | 0.47.1 | 2.1.23 &rarr; 3.0.4 | 2.0.0 &rarr; 2.0.3 | ✔️ (compatible SLF4J upgrade) |
| 1.2.4.1 | 0.46.1 | 2.1.21 &rarr; 2.1.23 | 1.7.32 &rarr; 1.7.36  | ✔️ (compatible SLF4J upgrade) |
| 1.2.4.2 | 0.46.1 | 2.1.21 &rarr; 3.0.4 | 1.7.32 &rarr; 2.0.3 | ❌ |
| 1.2.5-SNAPSHOT | 0.46.1 | 2.1.21 &rarr; 2.1.23 | 1.7.32 | ✔️ |

## Analysis

If we run `gradle dependencies` on the project where this issue occurs, we'll see:

```
diktat
+--- com.pinterest:ktlint:0.46.1
|    +--- com.pinterest.ktlint:ktlint-core:0.46.1
|    |    +--- org.jetbrains.kotlin:kotlin-compiler-embeddable:1.7.0 -> 1.7.10
|    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.7.10 -> 1.7.21
|    |    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.7.21
|    |    |    |    \--- org.jetbrains:annotations:13.0
|    |    |    +--- org.jetbrains.kotlin:kotlin-script-runtime:1.7.10
|    |    |    +--- org.jetbrains.kotlin:kotlin-reflect:1.7.10
|    |    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.7.10 -> 1.7.21 (*)
|    |    |    +--- org.jetbrains.kotlin:kotlin-daemon-embeddable:1.7.10
|    |    |    +--- org.jetbrains.intellij.deps:trove4j:1.0.20200330
|    |    |    \--- net.java.dev.jna:jna:5.6.0
|    |    +--- org.ec4j.core:ec4j-core:0.3.0
|    |    +--- io.github.microutils:kotlin-logging-jvm:2.1.21 -> 3.0.4
|    |    |    +--- org.slf4j:slf4j-api:2.0.3
|    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.20 -> 1.7.21
|    |    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.7.21 (*)
|    |    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.7.21
|    |    |    |         \--- org.jetbrains.kotlin:kotlin-stdlib:1.7.21 (*)
|    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib-common:1.7.20 -> 1.7.21
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 -> 1.7.21 (*)
|    +--- com.pinterest.ktlint:ktlint-reporter-baseline:0.46.1
|    |    +--- com.pinterest.ktlint:ktlint-core:0.46.1 (*)
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 -> 1.7.21 (*)
|    +--- com.pinterest.ktlint:ktlint-reporter-checkstyle:0.46.1
|    |    +--- com.pinterest.ktlint:ktlint-core:0.46.1 (*)
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 -> 1.7.21 (*)
|    +--- com.pinterest.ktlint:ktlint-reporter-json:0.46.1
|    |    +--- com.pinterest.ktlint:ktlint-core:0.46.1 (*)
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 -> 1.7.21 (*)
|    +--- com.pinterest.ktlint:ktlint-reporter-html:0.46.1
|    |    +--- com.pinterest.ktlint:ktlint-core:0.46.1 (*)
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 -> 1.7.21 (*)
|    +--- com.pinterest.ktlint:ktlint-reporter-plain:0.46.1
|    |    +--- com.pinterest.ktlint:ktlint-core:0.46.1 (*)
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 -> 1.7.21 (*)
|    +--- com.pinterest.ktlint:ktlint-reporter-sarif:0.46.1
|    |    +--- com.pinterest.ktlint:ktlint-core:0.46.1 (*)
|    |    +--- io.github.detekt.sarif4k:sarif4k:0.0.1
|    |    |    +--- org.jetbrains.kotlinx:kotlinx-serialization-json:1.1.0 -> 1.4.1
|    |    |    |    \--- org.jetbrains.kotlinx:kotlinx-serialization-json-jvm:1.4.1
|    |    |    |         +--- org.jetbrains.kotlin:kotlin-stdlib:1.7.20 -> 1.7.21 (*)
|    |    |    |         +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.7.20 -> 1.7.21
|    |    |    |         +--- org.jetbrains.kotlinx:kotlinx-serialization-core:1.4.1
|    |    |    |         |    \--- org.jetbrains.kotlinx:kotlinx-serialization-core-jvm:1.4.1
|    |    |    |         |         +--- org.jetbrains.kotlin:kotlin-stdlib:1.7.20 -> 1.7.21 (*)
|    |    |    |         |         +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.7.20 -> 1.7.21
|    |    |    |         |         \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.20 -> 1.7.21 (*)
|    |    |    |         \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.20 -> 1.7.21 (*)
|    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.4.31 -> 1.7.21 (*)
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 -> 1.7.21 (*)
|    +--- com.pinterest.ktlint:ktlint-ruleset-experimental:0.46.1
|    |    +--- com.pinterest.ktlint:ktlint-core:0.46.1 (*)
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 -> 1.7.21 (*)
|    +--- com.pinterest.ktlint:ktlint-ruleset-test:0.46.1
|    |    +--- com.pinterest.ktlint:ktlint-core:0.46.1 (*)
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 -> 1.7.21 (*)
|    +--- org.jetbrains.kotlin:kotlin-compiler-embeddable:1.7.0 -> 1.7.10 (*)
|    +--- com.github.shyiko.klob:klob:0.2.1
|    +--- info.picocli:picocli:4.6.3
|    +--- ch.qos.logback:logback-classic:1.2.11
|    |    +--- ch.qos.logback:logback-core:1.2.11
|    |    \--- org.slf4j:slf4j-api:1.7.32 -> 2.0.3
|    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 -> 1.7.21 (*)
\--- org.cqfn.diktat:diktat-rules:1.2.4.2
     +--- org.cqfn.diktat:diktat-common:1.2.4.2
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.21 (*)
     |    +--- org.jetbrains.kotlinx:kotlinx-serialization-json:1.4.1 (*)
     |    +--- com.charleskorn.kaml:kaml-jvm:0.47.0
     |    |    +--- org.jetbrains.kotlinx:kotlinx-serialization-core:1.3.3 -> 1.4.1 (*)
     |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.10 -> 1.7.21 (*)
     |    |    +--- org.jetbrains.kotlin:kotlin-bom:1.7.10
     |    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.7.10 -> 1.7.21 (c)
     |    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.7.10 -> 1.7.21 (c)
     |    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.10 -> 1.7.21 (c)
     |    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.7.10 -> 1.7.21 (c)
     |    |    |    +--- org.jetbrains.kotlin:kotlin-compiler-embeddable:1.7.10 (c)
     |    |    |    +--- org.jetbrains.kotlin:kotlin-script-runtime:1.7.10 (c)
     |    |    |    \--- org.jetbrains.kotlin:kotlin-reflect:1.7.10 (c)
     |    |    \--- org.snakeyaml:snakeyaml-engine:2.3
     |    +--- commons-cli:commons-cli:1.5.0
     |    +--- io.github.microutils:kotlin-logging-jvm:3.0.4 (*)
     |    \--- com.pinterest.ktlint:ktlint-core:0.46.1 (*)
     +--- com.pinterest.ktlint:ktlint-core:0.46.1 (*)
     +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.21 (*)
     +--- com.google.guava:guava:31.1-jre
     |    +--- com.google.guava:failureaccess:1.0.1
     |    +--- com.google.guava:listenablefuture:9999.0-empty-to-avoid-conflict-with-guava
     |    +--- com.google.code.findbugs:jsr305:3.0.2
     |    +--- org.checkerframework:checker-qual:3.12.0
     |    +--- com.google.errorprone:error_prone_annotations:2.11.0
     |    \--- com.google.j2objc:j2objc-annotations:1.3
     +--- io.github.microutils:kotlin-logging-jvm:3.0.4 (*)
     \--- com.bpodgursky:jbool_expressions:1.24
          \--- org.antlr:antlr-runtime:3.5.2
```

This is because (older) _KtLint_ **0.46.1** still has `slf4j-api` version **1.7.32** in its dependencies:

```
ktlint
\--- project :ktlint
     +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0
     |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.7.0
     |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.7.0
     |    |    \--- org.jetbrains:annotations:13.0
     |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.7.0
     |         \--- org.jetbrains.kotlin:kotlin-stdlib:1.7.0 (*)
     +--- project :ktlint-core
     |    +--- org.jetbrains.kotlin:kotlin-compiler-embeddable:1.7.0
     |    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.7.0 (*)
     |    |    +--- org.jetbrains.kotlin:kotlin-script-runtime:1.7.0
     |    |    +--- org.jetbrains.kotlin:kotlin-reflect:1.7.0
     |    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.7.0 (*)
     |    |    +--- org.jetbrains.kotlin:kotlin-daemon-embeddable:1.7.0
     |    |    +--- org.jetbrains.intellij.deps:trove4j:1.0.20200330
     |    |    \--- net.java.dev.jna:jna:5.6.0
     |    +--- org.ec4j.core:ec4j-core:0.3.0
     |    +--- io.github.microutils:kotlin-logging-jvm:2.1.21
     |    |    +--- org.slf4j:slf4j-api:1.7.32
     |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.6.0 -> 1.7.0 (*)
     |    |    \--- org.jetbrains.kotlin:kotlin-stdlib-common:1.6.0 -> 1.7.0
     |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 (*)
     +--- project :ktlint-reporter-baseline
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 (*)
     |    \--- project :ktlint-core (*)
     +--- project :ktlint-reporter-checkstyle
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 (*)
     |    \--- project :ktlint-core (*)
     +--- project :ktlint-reporter-json
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 (*)
     |    \--- project :ktlint-core (*)
     +--- project :ktlint-reporter-html
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 (*)
     |    \--- project :ktlint-core (*)
     +--- project :ktlint-reporter-plain
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 (*)
     |    \--- project :ktlint-core (*)
     +--- project :ktlint-reporter-sarif
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 (*)
     |    +--- project :ktlint-core (*)
     |    \--- io.github.detekt.sarif4k:sarif4k:0.0.1
     |         +--- org.jetbrains.kotlinx:kotlinx-serialization-json:1.1.0
     |         |    \--- org.jetbrains.kotlinx:kotlinx-serialization-json-jvm:1.1.0
     |         |         +--- org.jetbrains.kotlin:kotlin-stdlib:1.4.30 -> 1.7.0 (*)
     |         |         +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.4.30 -> 1.7.0
     |         |         \--- org.jetbrains.kotlinx:kotlinx-serialization-core:1.1.0
     |         |              \--- org.jetbrains.kotlinx:kotlinx-serialization-core-jvm:1.1.0
     |         |                   +--- org.jetbrains.kotlin:kotlin-stdlib:1.4.30 -> 1.7.0 (*)
     |         |                   \--- org.jetbrains.kotlin:kotlin-stdlib-common:1.4.30 -> 1.7.0
     |         \--- org.jetbrains.kotlin:kotlin-stdlib:1.4.31 -> 1.7.0 (*)
     +--- project :ktlint-ruleset-experimental
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 (*)
     |    \--- project :ktlint-core (*)
     +--- project :ktlint-ruleset-standard
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 (*)
     |    +--- project :ktlint-core (*)
     |    \--- io.github.microutils:kotlin-logging-jvm:2.1.21 (*)
     +--- project :ktlint-ruleset-test
     |    +--- project :ktlint-core (*)
     |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.0 (*)
     +--- org.jetbrains.kotlin:kotlin-compiler-embeddable:1.7.0 (*)
     +--- com.github.shyiko.klob:klob:0.2.1
     +--- info.picocli:picocli:4.6.3
     \--- ch.qos.logback:logback-classic:1.2.11
          +--- ch.qos.logback:logback-core:1.2.11
          \--- org.slf4j:slf4j-api:1.7.32
```

_KtLint_ **0.47.1**, on the other hand, upgrades `slf4j-api` from **1.7.32** to **2.0.0**:

```
ktlint
\--- project :ktlint
     +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.10
     |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.7.10
     |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.7.10
     |    |    \--- org.jetbrains:annotations:13.0
     |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.7.10
     |         \--- org.jetbrains.kotlin:kotlin-stdlib:1.7.10 (*)
     +--- project :ktlint-core
     |    +--- org.jetbrains.kotlin:kotlin-compiler-embeddable:1.7.10
     |    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.7.10 (*)
     |    |    +--- org.jetbrains.kotlin:kotlin-script-runtime:1.7.10
     |    |    +--- org.jetbrains.kotlin:kotlin-reflect:1.7.10
     |    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.7.10 (*)
     |    |    +--- org.jetbrains.kotlin:kotlin-daemon-embeddable:1.7.10
     |    |    +--- org.jetbrains.intellij.deps:trove4j:1.0.20200330
     |    |    \--- net.java.dev.jna:jna:5.6.0
     |    +--- org.ec4j.core:ec4j-core:0.3.0
     |    +--- io.github.microutils:kotlin-logging-jvm:2.1.23
     |    |    +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.6.0 -> 1.7.10
     |    |    +--- org.slf4j:slf4j-api:1.7.32 -> 2.0.0
     |    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.6.0 -> 1.7.10 (*)
     |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.10 (*)
     +--- project :ktlint-reporter-baseline
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.10 (*)
     |    \--- project :ktlint-core (*)
     +--- project :ktlint-reporter-checkstyle
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.10 (*)
     |    \--- project :ktlint-core (*)
     +--- project :ktlint-reporter-json
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.10 (*)
     |    \--- project :ktlint-core (*)
     +--- project :ktlint-reporter-format
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.10 (*)
     |    \--- project :ktlint-core (*)
     +--- project :ktlint-reporter-html
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.10 (*)
     |    \--- project :ktlint-core (*)
     +--- project :ktlint-reporter-plain
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.10 (*)
     |    \--- project :ktlint-core (*)
     +--- project :ktlint-reporter-sarif
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.10 (*)
     |    +--- project :ktlint-core (*)
     |    \--- io.github.detekt.sarif4k:sarif4k:0.0.1
     |         +--- org.jetbrains.kotlinx:kotlinx-serialization-json:1.1.0
     |         |    \--- org.jetbrains.kotlinx:kotlinx-serialization-json-jvm:1.1.0
     |         |         +--- org.jetbrains.kotlin:kotlin-stdlib:1.4.30 -> 1.7.10 (*)
     |         |         +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.4.30 -> 1.7.10
     |         |         \--- org.jetbrains.kotlinx:kotlinx-serialization-core:1.1.0
     |         |              \--- org.jetbrains.kotlinx:kotlinx-serialization-core-jvm:1.1.0
     |         |                   +--- org.jetbrains.kotlin:kotlin-stdlib:1.4.30 -> 1.7.10 (*)
     |         |                   \--- org.jetbrains.kotlin:kotlin-stdlib-common:1.4.30 -> 1.7.10
     |         \--- org.jetbrains.kotlin:kotlin-stdlib:1.4.31 -> 1.7.10 (*)
     +--- project :ktlint-ruleset-experimental
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.10 (*)
     |    \--- project :ktlint-core (*)
     +--- project :ktlint-ruleset-standard
     |    +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.10 (*)
     |    +--- project :ktlint-core (*)
     |    \--- io.github.microutils:kotlin-logging-jvm:2.1.23 (*)
     +--- project :ktlint-ruleset-test
     |    +--- project :ktlint-core (*)
     |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.7.10 (*)
     +--- org.jetbrains.kotlin:kotlin-compiler-embeddable:1.7.10 (*)
     +--- com.github.shyiko.klob:klob:0.2.1
     +--- info.picocli:picocli:4.6.3
     \--- ch.qos.logback:logback-classic:1.3.0
          +--- ch.qos.logback:logback-core:1.3.0
          \--- org.slf4j:slf4j-api:2.0.0
```

We need to:

* [ ] Add an integration test which would actually run the Gradle plug-in (probably as a part of #1569), and
* [ ] Upgrade the underlying KtLint to 0.47.1 and migrate to its new API (#1543).

## Environment information
* diktat version: **1.2.4.2**, _KtLint_ **0.46.1**
* build tool (maven/gradle): Gradle

**Comment by 0x6675636b796f75676974687562** [comment]
The issue does not affect Spotless 6.11.0 with Diktat 1.2.4.2 if the project is configured like this:

```kotlin
fun Project.configureSpotless() {
    val libs = the<LibrariesForLibs>()
    val diktatVersion = libs.versions.diktat.get()
    apply<SpotlessPlugin>()
    configure<SpotlessExtension> {
        kotlin {
            diktat(diktatVersion).configFile(rootProject.file("diktat-analysis.yml"))
            target("src/**/*.kt")
            if (path == rootProject.path) {
                target("buildSrc/**/*.kt")
            }
        }
        kotlinGradle {
            diktat(diktatVersion).configFile(rootProject.file("diktat-analysis.yml"))

            // using `Project#path` here, because it must be unique in gradle's project hierarchy
            if (path == rootProject.path) {
                target("$rootDir/*.kts", "$rootDir/buildSrc/**/*.kts")
            } else {
                target("**/*.kts")
            }
        }
    }
}
```

**Comment by nulls** [comment]
and it does work on maven plugin, because we have a test for it

**Comment by bddckr** [comment]
This exception is still raised on my project when using diktat via `org.cqfn.diktat.diktat-gradle-plugin:1.2.4.2` and running `diktatCheck`. However, version `1.2.4.1` works fine.

As explained above, using diktat via Spotless (`6.12.1`) works without issues.

Any ideas as to why I'm getting this issue when running `diktatCheck`?

**Comment by nulls** [comment]
hi, @bddckr,

We will look at the issue.

**Comment by 0x6675636b796f75676974687562** [comment]
@bddckr, the issue has been fixed with #1579 and will be available as a part of the upcoming 1.2.5 release.

We apologize for inconvenience.

**Comment by bddckr** [comment]
Thank you all for the work! Very much appreciated - looking forward to the release! 🚀
